### DISCUSSION TITLE
npx create-turbo@latest failing

### DISCUSSION THREAD

**Discussion by olayinkade** [vercel/turborepo#1843] [Help] [answered]
when I run npx create-turbo@latest, I go through all the processes needed to create my turborepo i get the below error message. I just wanted to confirm it is a bug versus me not doing something right

```
Unexpected error. Please report it as a bug:
TypeError: import_fs.default.cpSync is not a function
    at run (/Users/olayinka.adelakun/.npm/_npx/3089/lib/node_modules/create-turbo/dist/index.js:319:21)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)

```

**Comment by olayinkade** [comment, marked as answer]
It seems like the issue was with the node version. I was using node 14. I upgraded to 16 and it worked

**Reply by mehulkar** [reply]
We should probably define what versions we support somewhere! Since node 16 is the active LTS, it seems unlikely that we'll fix this for Node 14, but feel free to open an issue or make a PR!
